index.html

```diff
@@ -38,20 +38,50 @@
     <main>
       <section id="carousel">
         <div class="container">
-          <div class="carousel-list">
-            <div class="carousel-list__item">
-              <figure class="carousel-list__image">
-                <img src="https://placeimg.com/1000/400/arch" alt="Title" />
-              </figure>
-              <div class="carousel-list__info">
-                <span>Worldwide shipping</span>
-              </div>
+          <div class="carousel-wrap">
+            <div class="carousel-list">
+              <a href="" class="carousel-list__item">
+                <figure class="carousel-list__image">
```

Comment:
You could also consider using `figcaption` inside of `figure` to contain the description related to an item.